test/logging_test.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/xlab-si/emmy/log"
+	"testing"
+)
+
+func TestInvalidLogLevel(t *testing.T) {
+	_, err := log.NewStdoutLogger("test", "qwerty", log.FORMAT_SHORT)
+	assert.NotNil(t, err, "should produce an error due to invalid log level")
+}
+
+func TestInvalidFormat(t *testing.T) {
+	_, err := log.NewStdoutLogger("test", log.INFO, "blahblah")
```

Comment:
`"blahblah"` could be replaced with something more descriptive, say `"invalid-format"`

Reply:
:D Sorry, got carried away. Okay, done.